drivers/project_scorecard/app/jobs/project_scorecard/populate_scorecard.rb

```diff
@@ -0,0 +1,179 @@
+###
+# Copyright 2016 - 2020 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module ProjectScorecard
+  class PopulateScorecard < BaseJob
```

Comment:
I think this should use the generic report job similar to how the pass/fail report works.  Logic should be in the model, not the job.